modules/programs/kakoune.nix

```diff
@@ -0,0 +1,342 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.kakoune;
+
+  configModule = types.submodule {
+    options = {
+
+      tabStop = mkOption {
+        type = types.ints.unsigned;
+        default = 8;
+        description = ''
+          The width of a tab
```

Comment:
All option description should end in a period. I.e., here it should be `The width of a tab.`.